### PR TITLE
Add beginner's guide as a reusable page

### PR DESCRIPTION
**Pull request by binford2k**
This also adds it to the navbar, I'm assuming that's where we want it
linked. If it's too busy, we could probably turn off the CFP link for
now?

### CODE REVIEW

File: content/ghent2025/guide.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        fruits, pastries, and such. And there will be a lot of people excited to talk to
+        each other. This is where most of the hallway track happens. When you’re done
+        with a talk and you don’t have anything in particular to do, you should come
+        here. If you are in the lobby and it seems particularly empty, then there’s
+        probably a talk happening. If you didn’t expect that, ask someone. There may
+        have been a schedule change that didn’t get published.
+
+    1. **Swag pickup**
+
+        Walter will have to do this section 😀
```
Comment (binford2k) [inline content/ghent2025/guide.md:109]: @walterheck this is for you

File: content/ghent2025/guide.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    down to baggage claim and you’ll find vending machines where you can purchase
+    your train ticket to the **Gent-Sint-Pieters** station. From there, see #3 for
+    local transportation.
+
+    Do not assume that you can stay in Brussels and commute to Ghent. You’ll spend a
+    lot of time on the train and you’ll miss all the social events.
+
+    1. **Coming from Fosdem**
+
+        If you’re coming from Fosdem and are traveling light, then you may be interested
```
Comment (binford2k) [inline content/ghent2025/guide.md:36]: @KrisBuytaert you ok with this?

Comment (KrisBuytaert) [reply content/ghent2025/guide.md:36]: Maybe point to me personally but to the CfgMgmtCamp socials ..

File: content/ghent2025/guide.md (hunk 1 above)
Comment (walterheck) [reply content/ghent2025/guide.md:109]: Pickup on Monday before the keynotes in the hall where the sponsors are. Foe questions email me at walterheck@helixiora.com

**Pull request merged**